api/v1beta1/connector_constants.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+const ComponentTypeConnector = "connector"
+
+type ConnectorType string
+
+// +kubebuilder:validation:Enum=sink-elasticsearch
```

Comment:
I'm not sure is this work ?

Reply:
Looks it is not work, because the enum is not in crds.yaml

Reply:
Yes, I should have put it above the spec fields. Fixed in [e68a822](https://github.com/hstreamdb/hstream-operator/pull/59/commits/e68a822351779e62ad6e165e22cf255bb36b4a08).